app/views/steps/petition/orders/edit.html.erb

```diff
@@ -6,33 +6,17 @@
 
     <h1 class="heading-xlarge gv-u-heading-xxlarge"><%=t '.heading' %></h1>
 
-    <p class="lede gv-u-text-lede"><%=t '.lead_text' %></p>
-
     <%= step_form @form_object do |f| %>
-      <%= f.check_box_fieldset :orders, PetitionOrder::CHILD_ARRANGEMENTS %>
-
       <%=
-        f.check_box_fieldset :orders, [
-          PetitionOrder::PROHIBITED_STEPS,
-        ] do |fieldset|
+        f.check_box_fieldset :orders, [], legend_options: { class: 'visually-hidden', text: t('.heading') } do |fieldset|
+          fieldset.check_box_input(PetitionOrder::CHILD_ARRANGEMENTS_HOME)
```

Comment:
Is there a way of flattening this all out so that we're not rendering more `<fieldset>` elements? I tried replacing the `fieldset.check_box_input` with `f.check_box_input` but that wasn't quite enough

Reply:
No. It is the same in the MIAM exemptions.

There is a top level fieldset, and inside there are as many other fieldsets as groups of checkbox we have (those ones that open when you check a checkbox).

So here we have 2 groups, there will be 2 fieldsets (one for `orders_prohibited_steps` and another one for `orders_specific_issues`), inside the top level fieldset.

It is what creates the "group" of checkboxes. And we have 3. One top level with the first-level checkboxes, and then 2 sub-groups.